ui/browser/views/browser.jsx

```diff
@@ -34,21 +37,39 @@ class BrowserWindow extends Component {
   }
 
   render() {
-    const { pages, currentPageIndex, pageOrder, ipcRenderer } = this.props;
+    const { dispatch, pages, currentPageIndex, pageOrder, ipcRenderer } = this.props;
     const platformClass = `platform-${platform}`;
+    const navBack = e => getCurrentWebView(e.target.ownerDocument).goBack();
```

Comment:
Any reason that some of these are defined above the jsx and others are in line (like bookmark)?